Vox Away Site Overhaul

bigfunguss opened the pull request:
:cl: Funguss, Imienny, Cheb Pomidorov, and MyphicBowser
rscadd: Includes earlier PR by Imienny - Added a new roles to Vox Scavenger away site, "Shoal Biotechnician", "Shoal Technician" and "Quill"
maptweak: New, functional vox ship, with multiple departments to accomodate the new roles, and a working (And re-docking) shuttle, initial outline by MyphicBowser, coded by Cheb, and mapped by Funguss.
/:cl:

Demo image of the ship: https://i.imgur.com/SFUYphF.png

bigfunguss committed:
Vox map and job overhaul.

New large ship and docked shuttle in place of the old meteor crash. Three new jobs: Quill, Biotechnician, and Shoal Technician courtesy of Imienny.

bigfunguss committed:
Delete voxship-1.dmm

Replaced by the new map

bigfunguss committed:
Vox Map Overhaul

(This time with bug fixes by Cheb Pomidorov)

bigfunguss committed:
Fixed duplicate catwalk

Double catwalk was creating a warning.

Roland410 commented on maps/away/voxship/voxship_jobs.dm:
```diff
@@ -1,30 +1,165 @@
+	whitelisted_species = list(SPECIES_VOX)
+	blacklisted_species = null
+	is_semi_antagonist = TRUE
+	max_skill = list(   SKILL_BUREAUCRACY = SKILL_MAX,
```
I've removed the cap on max_skills for away site people, so you'd only have to override those you want to be less than max both here and below.

Roland410 commented:
You also seem to have accidentally modified the transit level? On the other hand, not sure which one of you to ping for feedback, but don't think they need that many guns even if they are provocateurs.

bigfunguss commented:
The transit level change was for the shuttle placement afaik, Cheb did all of the coding for that, I just did design and mapping on voxship.dmm

Soapenhauer commented:
Hello, I modified z2_transit.dmm, to correct the shuttle's outline and ensure no one gets yeeted out of the shuttle and into space, during transit.

bigfunguss committed:
Vox Overhaul

-Nerfed number of randomized weapons in the armory to four, and one one random handgun elsewhere.  Added coats and RNG hats in their place for flavor.
-Made minor tweaks to other departments on the new ship, medbay has a vitals analyzer by the operating table, hydroponics has a seed storage, and the atmos gas tanks are now color coded
-Fixed a few floor tiles that weren't rendering correctly
-The jobs file was modified by Cheb based on maintainer advice

bigfunguss committed:
Merge branch 'dev' into dev

bigfunguss committed:
Travis troubleshooting

removed a blood drip that caused a Travis fail

bigfunguss committed:
Voxship Jobs

re-committing skill max changes by Roland that were overwritten

bigfunguss committed:
Vox Ship minor fixes

-Atmos tweaked, creates much less waste and uses less power
-Ore refinery inputting and outputting in the correct direction again.
-Random hats removed from the armory, just defaulted to hardhats most of the time, ended up being pointless.

bigfunguss committed:
Removing files to allow master to merge into my fork

bigfunguss committed:
Ghetto manual merge

attempting to sort out problems with auto merge

bigfunguss committed:
Delete voxship-1.dmm

attempt at an auto merge fix

bigfunguss committed:
Vox Ship Overhaul

Re-committing the updated files to smooth over automerge troubles
